### ISSUE TITLE
add support for enums

### ISSUE THREAD

**Issue by vanlooverenkoen** [icapps/flutter-model-generator#19]
Add support for enums. This is currently not supported

**Comment by vanlooverenkoen** [comment]
```
Gender:
  path: user/person/
  type: enum
  properties:
    MALE:
      value: _mAl3
    FEMALE:
      value: femAle
    X:
      value: X
```

We will use this type of definition-

**Comment by vanlooverenkoen** [comment]
Fixed in 1.4.0
